modules/staging.nix

```diff
@@ -11,7 +11,7 @@ with import ./../lib.nix;
     };
 
     services = {
-      dd-agent.tags              = ["env:rc-staging"];
+      dd-agent.tags              = ["env:staging"];
```

Comment:
At the moment it's useful having both `rc-staging` and `staging` tags in DD. I'd like to keep it that way, unless we will kill the existing `staging` deployment before deploying this change to `rc-staging`.

Reply:
fixed, we now have a proper `depl:` tag